packages/@misk/simpleredux/src/utilities/simpleSelector.ts

```diff
@@ -41,7 +41,7 @@ const selectRawSubState: <
 
 const rawSubStateSelector: <IState extends Map<string, any>, ISubState>(
   domain: string
-) => OutputSelector<IState, any, (res: ISubState) => any> = <
+) => OutputSelector<any, any, (res: ISubState) => any> = <
```

Comment:
These errors were really difficult to parse, though I'm not sure if any behaviour is actually changing, I'll include it below:

```
ERROR in /Users/krisp/Development/misk-web/packages/@misk/simpleredux/src/utilities/simpleSelector.ts
./src/utilities/simpleSelector.ts
[tsl] ERROR in /Users/krisp/Development/misk-web/packages/@misk/simpleredux/src/utilities/simpleSelector.ts(95,59)
      TS2345: Argument of type 'IState' is not assignable to parameter of type 'GetStateFromSelectors<IState>'.
  Type '{ [key: string]: any; }' is not assignable to type 'RemoveNames<{ [index in keyof LongestArray<TuplifyUnion<Transpose<ExtractParams<IState>>, LastOf<Transpose<ExtractParams<IState>>>, [Transpose<ExtractParams<...>>] extends [...] ? true : false>>]: LongestArray<...>[index] extends LongestArray<...>[number] ? IgnoreInvalidIntersections<...> : never; }>[0] extends Remo...'.
```